DataFormats/JetReco/interface/GenJet.h

```diff
@@ -62,6 +94,19 @@ class GenJet : public Jet {
   /** Returns other energy (undecayed Sigmas etc.)*/
   float auxiliaryEnergy() const {return m_specific.m_AuxiliaryEnergy;};
 
+
+  // PF-like definitions
+  float chargedHadronEnergy() const { return m_specific.mChargedHadronEnergy;}
+  float neutralHadronEnergy() const { return m_specific.mNeutralHadronEnergy;}
+  float chargedEmEnergy    () const { return m_specific.mChargedEmEnergy    ;}
+  float neutralEmEnergy    () const { return m_specific.mNeutralEmEnergy    ;}
+  float muonEnergy         () const { return m_specific.mMuonEnergy         ;}
+  int chargedHadronMultiplicity() const { return m_specific.mChargedHadronMultiplicity;}
+  int neutralHadronMultiplicity() const { return m_specific.mNeutralHadronMultiplicity;}
+  int chargedEmMultiplicity    () const { return m_specific.mChargedEmMultiplicity    ;}
```

Comment:
in addition to charged EM and neutral EM, the PFJets provide electron and photon (which are basically the same, I think). Maybe these could be added here for consistency?

Reply:
I was of two minds about this, but it's probably more confusing than it's worth. "Charged" and "Neutral" EM are the same as "electron" and "photon" in the central region but differ in the forward region. For sure we shouldn't add separate data members, the information would be redundant. I could add accessors that are identical, but that seemed more confusing to me. 

I can do the rebase though.